auth0/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "3.23.1"
+__version__ = "3.23.2"
```

Comment:
```suggestion
__version__ = "3.23.1"
```

This gets updated when we do a release